src/main/java/opendota/Parse.java

```diff
@@ -146,7 +151,15 @@ public UnknownItemFoundException(String message) {
 	private GreevilsGreedVisitor greevilsGreedVisitor;
 	private TrackVisitor trackVisitor;
     private ArrayList<Boolean> isPlayerStartingItemsWritten;
-    
+
+    //Draft stage variable
+    Integer draftStage = 0;
```

Comment:
I don't think draftStage needs to global either, you can assign/check its value in onTick